pretyping/reductionops.ml

```diff
@@ -1138,8 +1138,7 @@ let sigma_check_inductive_instances cv_pb variance u1 u2 sigma =
 
 let sigma_univ_state =
   let open Reduction in
-  { compare_graph = Evd.universes;
-    compare_sorts = sigma_compare_sorts;
+  { compare_sorts = sigma_compare_sorts;
```

Comment:
Not sure how much it matters, but this change is incorrect since after you `compare_sorts` the separately passed graph is out of date.

Reply:
Actually, I am not sure I understand. In what call path do we first `compare_sorts` and then call conversion again? All entrypoints in `Reduction` start by evaluating the graph.

Reply:
You're right, it's wrong in master too (the possibly outdated graph from the infos is used by the callback when reducing uip matches).
I'm OK with the PR then.

Reply:
Erm, isn't that a bug then? Or does it matter at all?

Reply:
It's a questionable behaviour but probably doesn't matter.

Reply:
:shrug: